### PR TITLE
Telegram Bot API 3.1 Update

### PR DESCRIPTION
**Pull request by Olfi01**
Have fun fixing the differences.

### CODE REVIEW

**Comment by Olfi01** [comment]
Actually resolving will be rather easy :smile:

**Comment by Olfi01** [comment]
@MrRoundRobin i did that for you xD hope nuget now gets updated soon

**Comment by Olfi01** [comment]
* `SetChatDescriptionAsync`

**Comment by Olfi01** [comment]
urgh
i should stop programming out of boredom

**Comment by Olfi01** [comment]
the f is with that commit title... welp i resolved the conflicts, who cares

File: src/Telegram.Bot/Converters/UnixDatetimeConverter.cs [hunk 1]
```diff
@@ -26,7 +26,7 @@ public override void WriteJson(JsonWriter writer, object value, JsonSerializer s
             long val;
             if (value is DateTime || value is Nullable<DateTime>)
             {
-                val = ((DateTime)value).ToUnixTime();
+                val = ((DateTime)value).ToUniversalTime().ToUnixTime();
```
Comment (MrRoundRobin) [inline src/Telegram.Bot/Converters/UnixDatetimeConverter.cs]: That's already done in `ToUnixTime`

File: src/Telegram.Bot/TelegramBotClient.cs [hunk 2]
```diff
@@ -165,7 +166,7 @@ public TelegramBotClient(string token, HttpClient httpClient = null)
             _token = token;
             _httpClient = httpClient ?? new HttpClient();
         }
-
+
```
Comment (deril) [inline src/Telegram.Bot/TelegramBotClient.cs]: Useless spaces

File: src/Telegram.Bot/TelegramBotClient.cs [hunk 3]
```diff
@@ -186,8 +187,7 @@ public TelegramBotClient(string token, IWebProxy webProxy)
             _token = token;
             _httpClient = new HttpClient(httpClientHander);
         }
-
-
+
```
Comment (deril) [inline src/Telegram.Bot/TelegramBotClient.cs]: Useless spaces

File: src/Telegram.Bot/Types/ChatPhoto.cs [hunk 4]
```diff
@@ -0,0 +1,25 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Text;
```
Comment (deril) [inline src/Telegram.Bot/Types/ChatPhoto.cs]: You don't need
`using System;
using System.Collections.Generic;
using System.Text;`
for this class

File: src/Telegram.Bot/Converters/UnixDatetimeConverter.cs (hunk 1 above)
Comment (Olfi01) [inline src/Telegram.Bot/Converters/UnixDatetimeConverter.cs]: is it? sorry, my bad

File: src/Telegram.Bot/TelegramBotClient.cs (hunk 2 above)
Comment (Olfi01) [inline src/Telegram.Bot/TelegramBotClient.cs]: visual studios fault, not gonna meddle with that

File: src/Telegram.Bot/Types/ChatPhoto.cs (hunk 4 above)
Comment (Olfi01) [inline src/Telegram.Bot/Types/ChatPhoto.cs]: ya

File: src/Telegram.Bot/TelegramBotClient.cs (hunk 3 above)
Comment (Olfi01) [inline src/Telegram.Bot/TelegramBotClient.cs]: again, VS fault.

File: src/Telegram.Bot/Converters/UnixDatetimeConverter.cs (hunk 1 above)
Comment (Olfi01) [inline src/Telegram.Bot/Converters/UnixDatetimeConverter.cs]: Fixed.

**Comment by MrRoundRobin** [comment]
@Olfi01 I squash merged this manualy as there are so many commits and reverts of changes, I think this is much cleaner.

beta is also uploaded to nuget

**Comment by Olfi01** [comment]
okay :)

**Pull request closed without merging**